packages/components/src/components/PackageActions.tsx

```diff
@@ -107,7 +95,10 @@ export const PackageActions: React.FC<PackageActionsProps> = (props) => {
           {dependencyString}
         </Button>
 
-        <PackageDependantsLink community={communityName} package={packageName}>
+        <PackageDependantsLink
+          community={communityIdentifier}
+          package={packageName}
```

Comment:
This is probably lacking namespace (unless it's implied in `packageName` alread, which I doubt). I'm expecting this might be addressed with the upcoming data provider PR (due to the backend also needing it), so if it's easier to address at the same time there, feel free to ignore for now in this PR.

Reply:
This was actually discussed in [a backend PR](https://github.com/thunderstore-io/Thunderstore/pull/507) but I'll comment here also for completeness sake: namespacing was overlooked altogether in the links and will be addressed in a separate PR.